repos/public-user/config/services.config.php

```diff
@@ -22,6 +25,27 @@
                 );
             },
 
+            AuthSettings::class => function (ContainerInterface $container) {
+                // @todo hook up to actual settings.
+                return new AuthSettings(
+                    AuthSettings::EXAMPLE_SETTINGS['clients'],
+                    AuthSettings::EXAMPLE_SETTINGS['scopes']
```

Comment:
I don't think the scope allows for this to be editable in a UI, but we could allow this to be part of the `application.config.php` and source it from there. I will hook that up before merging.